include/tfm/sdfw_psa_ipc_service.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef __SDFW_PSA_IPC_SERVICE_H__
+#define __SDFW_PSA_IPC_SERVICE_H__
+
+/*
+ * This header contains symbols that are used by both the SDFW client
+ * and the SDFW service.
+ */
+enum {
+	INDEX_HANDLE,
+	INDEX_IN_VEC,
+	INDEX_IN_LEN,
+	INDEX_OUT_VEC,
+	INDEX_OUT_LEN,
+	INDEX_STATUS_PTR,
```

Comment:
users won't be including this file directly, right (in which case these should probably be namespaced a bit more)?

Reply:
I'm starting to miss the old 0, 1, 2, 3 XD

ipc_service_buf[0]
vs
ipc_service_buf[SDFW_PSA_IPC_SERVICE_INDEX_STATUS_PTR]

Reply:
I'm fine with either, as long as the use of the magic numbers is clear and centralized to one location (which they are in this case)

Reply:
added prefix